examples/00-load/create-pixel-art.py

```diff
@@ -0,0 +1,96 @@
+"""
+.._ref_geometric_example:
+
+Pixel Art of ALIEN MONSTERS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here we use :func:`pyvista.Box` to make pixel art.
+https://en.wikipedia.org/wiki/Pixel_art
+Source of characters:
+https://commons.wikimedia.org/wiki/File:Noto_Emoji_Pie_1f47e.svg
+License:
+https://github.com/googlefonts/noto-emoji#license
+"""
+import pyvista as pv
+from pyvista.demos import logo
+
+###############################################################################
+# Generate Piexls of each characters
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# You can define the pixel of INVADERS
+
+#       %         %
+#         %     %
+#       % % % % % %
+#     % %   % %   % %
+#   % % % % % % % % % %
+#   %   % % % % % %   %
+#   %   %         %   %
+#   %   % %     % %   %
+#         %     %
+#       %         %
+
+
+alien = [
+    [False, False, True , False, False, False, False, True , False, False],
+    [False, False, False, True , False, False, True , False, False, False],
+    [False, False, True , True , True , True , True , True , False, False],
+    [False, True , True , False, True , True , False, True , True , False],
+    [True , True , True , True , True , True , True , True , True , True ],
+    [True , False, True , True , True , True , True , True , False, True ],
+    [True , False, True , False, False, False, False, True , False, True ],
+    [True , False, True , True , False, False, True , True , False, True ],
+    [False, False, False, True , False, False, True , False, False, False],
+    [False, False, True , False, False, False, False, True , False, False],
+]
```

Comment:
This can be done programmatically:

```suggestion
alien_str = """
    %         %
      %     %
    % % % % % %
  % %   % %   % %
% % % % % % % % % %
%   % % % % % %   %
%   %         %   %
%   % %     % %   %
      %     %
    %         %
"""


alien = []
for line in alien_str.splitlines()[1:]:  # skip first linebreak
    if not line:
        continue
    if len(line) < 20:
        line += (20 - len(line))*' '
    alien.append([line[i:i + 2] == '% ' for i in range(0, len(line), 2)])

```

Reply:
We could also remove the whitespace borders altogether, convert to a numpy array, and use a single `== '%'` test, I think :)

Reply:
I had considered that as well and was going for a pretty string first and foremost.

If we're going for true purism, we'd use a method to parse the original pixel art into an array, but the point was to show how we can get a string as a simple mesh.